src/app/static/src/app/components/forms/DynamicFormSelect.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
I kept this separate from the multiselect because it seemed like overkill to add the treeselect widget when a normal html5 select would do. I have done some work to style them the same though.